calico_kubernetes/calico_kubernetes.py

```diff
@@ -311,10 +345,10 @@ def _generate_profile_json(self, profile_name, rules):
             'outbound_rules': outbound,
         }
         profile_json = json.dumps(profile, indent=2)
-        print('Final profile "%s": %s' % (profile_name, profile_json))
+        print('Final profile "%s":\n%s' % (profile_name, profile_json))
         return profile_json
 
-    def _apply_rules(self, profile_name):
+    def _apply_rules(self, profile_name, pod):
         """
         Generate a new profile with the default 'allow all' rules.
```

Comment:
Is this really what this method does?

Looks more like it takes an existing profile, removes the default rules, and then applies the correct inbound and outbound rules as generated in _generate_rules.